lib/feature_flagger/storage/feature_keys_migration.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module FeatureFlagger
+    module Storage
+      class FeatureKeysMigration
+
+        def initialize(from_redis, to_control)
+          @from_redis = from_redis
+          @to_control = to_control
+        end
+
+        # call migrates features key from the old fashioned to the new
+        # format.
+        #
+        # It must replicate feature keys with changes:
+        #
+        # from "avenue:traffic_lights" => 42
+        # to   "avenue:42" => traffic_lights
+        def call
+          @from_redis.scan_each(match: "*", count: FeatureFlagger::Storage::Redis::SCAN_EACH_BATCH_SIZE) do |redis_key|
+            # filter out resource_keys
+            next if redis_key.start_with?("#{FeatureFlagger::Storage::Redis::RESOURCE_PREFIX}:")
```

Comment:
I didn't find a test that covered this case here. As it appears to be a very critical issue it would be interesting to be covered by tests